### PR TITLE
[dynamo][benchmarks] HF - Fix seq len and batch sizes

### PR DESCRIPTION
**Pull request by anijain2305**
cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire @ngimel 

Fixes many models in https://github.com/pytorch/torchdynamo/issues/1842

### CODE REVIEW

File: benchmarks/dynamo/huggingface.py [hunk 1]
```diff
@@ -89,42 +89,68 @@ def pip_install(package):
...
+    # "GPTNeoForCausalLM" : 2,
+    # "GPTNeoForSequenceClassification" : 2,
+    "GoogleFnet": 2,
     "LayoutLMForMaskedLM": 2,
     "LayoutLMForSequenceClassification": 2,
+    "M2M100ForConditionalGeneration": 4,
+    "MBartForCausalLM": 2,
+    "MBartForConditionalGeneration": 2,
+    "MT5ForConditionalGeneration": 2,
+    "MegatronBertForCausalLM": 4,
```
Comment (ngimel) [inline benchmarks/dynamo/huggingface.py:136]: That's weird, it makes bs=4 (16/4), but I ran it with bs=12 even with cudagraphs.

File: benchmarks/dynamo/huggingface_models_list.txt [hunk 2]
```diff
@@ -1,53 +1,51 @@
 AlbertForMaskedLM,8
 AlbertForQuestionAnswering,8
-AllenaiLongformerBase,1
-BartForCausalLM,16
+AllenaiLongformerBase,8
```
Comment (ngimel) [inline benchmarks/dynamo/huggingface_models_list.txt:3]: so sizes in this file are maximum size for eager?

**Review by ngimel — APPROVED** [review]

File: benchmarks/dynamo/huggingface_models_list.txt (hunk 2 above)
Comment (anijain2305) [reply benchmarks/dynamo/huggingface_models_list.txt:3]: Yes

File: benchmarks/dynamo/huggingface.py (hunk 1 above)
Comment (anijain2305) [reply benchmarks/dynamo/huggingface.py:136]: It could be because I changed the seq_length

Comment (ngimel) [reply benchmarks/dynamo/huggingface.py:136]: I tried with a longer seq length :-)

Comment (anijain2305) [reply benchmarks/dynamo/huggingface.py:136]: Discussed offline - I was trying float32, while Natalia was trying AMP. For now, we are going to stick with float32 and revisit if the results still look bad.

**Pull request closed without merging**